From: Geert Uytterhoeven <Geert.Uytterhoeven@sonycom.com>

VT_HW_CONSOLE_BINDING must be enabled to make console unbinding work

Signed-off-by: Geert Uytterhoeven <Geert.Uytterhoeven@sonycom.com>
---
 drivers/video/Kconfig |    1 +
 1 files changed, 1 insertion(+)

--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -1796,6 +1796,7 @@ config FB_PS3
 	select FB_SYS_COPYAREA
 	select FB_SYS_IMAGEBLIT
 	select FB_SYS_FOPS
+	select VT_HW_CONSOLE_BINDING if FRAMEBUFFER_CONSOLE
 	---help---
 	  Include support for the virtual frame buffer in the PS3 platform.
 
